### PR TITLE
Add test for `RelativeCountTrendChartTest`

### PR DESCRIPTION
**Pull request by NikolasFunction**
Added Test for class RelativeCountTrendChart.java
JENKINS-66866

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/test/java/io/jenkins/plugins/forensics/miner/RelativeCountTrendChartTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import edu.hm.hafner.echarts.SeriesBuilder;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Tests RelativeCountTrendChart.
+ *
+ * @author Nikolas Paripovic
+ *
```
Comment (uhafner) [inline src/test/java/io/jenkins/plugins/forensics/miner/RelativeCountTrendChartTest.java:21]:
```suggestion
```

File: src/test/java/io/jenkins/plugins/forensics/miner/RelativeCountTrendChartTest.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+ */
+class RelativeCountTrendChartTest {
+
```
Comment (uhafner) [inline src/test/java/io/jenkins/plugins/forensics/miner/RelativeCountTrendChartTest.java:24]: Use new lines only to structure code
```suggestion
```

File: src/test/java/io/jenkins/plugins/forensics/miner/RelativeCountTrendChartTest.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    private final RelativeCountTrendChart relativeCountTrendChart = new RelativeCountTrendChart();
+
+    @Test
+    void shouldCreate() {
+
```
Comment (uhafner) [inline src/test/java/io/jenkins/plugins/forensics/miner/RelativeCountTrendChartTest.java:29]:
```suggestion
```

File: src/test/java/io/jenkins/plugins/forensics/miner/RelativeCountTrendChartTest.java [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+/**
+ * Tests RelativeCountTrendChart.
+ *
+ * @author Nikolas Paripovic
+ *
+ */
+class RelativeCountTrendChartTest {
+
+    private final RelativeCountTrendChart relativeCountTrendChart = new RelativeCountTrendChart();
```
Comment (uhafner) [inline src/test/java/io/jenkins/plugins/forensics/miner/RelativeCountTrendChartTest.java:25]: No fields in unit tests, inline that into the method

File: src/test/java/io/jenkins/plugins/forensics/miner/RelativeCountTrendChartTest.java [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+    @Test
+    void shouldCreate() {
+
+        Iterable<BuildResult<CommitStatisticsBuildAction>> buildResult = createBuildResults(false);
```
Comment (uhafner) [inline src/test/java/io/jenkins/plugins/forensics/miner/RelativeCountTrendChartTest.java:30]: Inline that call to make it readable:
```suggestion
        Iterable<BuildResult<CommitStatisticsBuildAction>> buildResult = new ArrayList<>();
```

File: src/test/java/io/jenkins/plugins/forensics/miner/RelativeCountTrendChartTest.java [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+    @Test
+    void shouldCreateWithData() {
+
+        Iterable<BuildResult<CommitStatisticsBuildAction>> buildResult = createBuildResults(true);
+        ChartModelConfiguration chartModelConfiguration = createChartModelConfiguration();
+        SeriesBuilder<CommitStatisticsBuildAction> seriesBuilder = createSeriesBuilder();
+
+        LinesChartModel linesChartModel = relativeCountTrendChart.create(buildResult, chartModelConfiguration, seriesBuilder);
+        assertThat(linesChartModel.getSeries()).hasSize(3);
+        assertThat(linesChartModel.getSeries()).allSatisfy(series -> assertThat(series.getData()).hasSize(4));
```
Comment (uhafner) [inline src/test/java/io/jenkins/plugins/forensics/miner/RelativeCountTrendChartTest.java:47]: Optional: if you want to make sure that the chart actually is correct, you can use JSON assertions, see review in https://github.com/jenkinsci/forensics-api-plugin/pull/351

**Review by uhafner — CHANGES_REQUESTED** [review]

**Comment by Fiesepille** [comment]
Seems to be a bug that commit/build #73bcd5d failed.
Following build #404805e succeeded, allthough the code styling was the only change.

**Comment by uhafner** [comment]
> Seems to be a bug that commit/build #73bcd5d failed.
> Following build #404805e succeeded, allthough the code styling was the only change.

I think the download of an artifact from the maven repository was broken but that should be fixed now.

**Review by uhafner — APPROVED** [review]

**Pull request merged**